layouts/fastly/helix.vcl

```diff
@@ -557,19 +557,26 @@ sub hlx_type_cgi {
   set var.namespace = regsuball(req.http.X-Action-Root, "/.*$", ""); // cut away the slash and everything after it
   set var.package = regsuball(req.http.X-Action-Root, "^.*/", ""); // cut away everything from the start up to (including) the slash
 
+  # Load important information from edge dicts
+  call hlx_github_static_owner;
+  call hlx_github_static_repo;
+  call hlx_github_static_ref;
 
   set var.script = regsuball(req.url.basename, "\..*$", "");
 
   # get (strain-specific) parameter whitelist
   include "params.vcl";
 
   set req.http.X-Backend-URL = "/api/v1/web"
-    + "/" + var.namespace // i.e. /trieloff
+    + "/" + var.namespace
     + "/" + var.package
     // looks like cgi-bin-hello-world for /cgi-bin/hello-world.js
     + "/cgi-bin-" + var.script
-    // we keep the complete query string
-    + "?" + req.url.qs;
+    + "?__hlx_owner=" + req.http.X-Github-Static-Owner
```

Comment:
why static?

Reply:
If it should be `code` instead, I don't see it available in the Fastly tables...